tools/templates/moar/Makefile.in

```diff
@@ -92,7 +92,7 @@ R_SETTING_MOAR = RESTRICTED.setting.moarvm
 @bpv(HARNESS6)@ = @nfp(@base_dir@/@bpm(RUNNER)@)@ -Ilib @nfp(t/harness6)@
 @bpv(HARNESS6_WITH_FUDGE)@ = @bpm(HARNESS6)@ --fudge
 
-@bpv(RUN_CLEAN_TARGET_FILES)@ = @bpm(RUN_RAKUDO)@ -e "for @*ARGS.head(*-1) { given (@*ARGS[*-1] ~ '.'.IO.SPEC.dir-sep ~ .IO.basename.Str) { say 'rm -f ' ~ .Str if '@verbose@' eq '@'; .IO.unlink if .IO.e } }"
+@bpv(RUN_CLEAN_TARGET_FILES)@ = @bpm(RUN_RAKUDO)@ @nfp(@base_dir@/tools/clean-target-files.raku
```

Comment:
All build/install scripts must go into _tools/build_ directory. `clean-target-files.raku` should be wrapped with `@script()@` macro which puts the directory and correct dirseps into place.

Reply:
Done.